docs/contribute/contribution-guidelines.md

```diff
@@ -32,21 +31,42 @@ Then link your local repository to your newly created fork.
 $ git remote add myprysmrepo https://github.com/<your_github_user_name>/prysm.git
 ```
 
-Finally, ensure bazel is installed and working on your machine:
+Finally, ensure Go is installed and working on your machine:
 
 ```text
-$ bazel version
+$ go version
 ```
 
-### Setting up your code / text editor
+Example output:
+```text
+go version go1.14.4 darwin/amd64
+```
 
-For developing Prysm, the only option is to use the `bazel` tool - Prysm _cannot_ be built with Go by itself. In order to write code for the Prysm codebase comfortably, we recommend using either [Visual Studio Code](https://code.visualstudio.com/download) with its [Bazel plugin](https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel), or any [Jetbrains IDE](https://www.jetbrains.com/) with the [Bazel plugin](https://plugins.jetbrains.com/plugin/8609-bazel) ([Goland](https://www.jetbrains.com/go/) is a great choice, used by most of the Prysmatic Labs team). 
+### Building and testing Prysm with Go
 
-You can also find various other types of IDE support for Bazel in the official Bazel documentation [here](https://docs.bazel.build/versions/master/ide.html). Once you have your coding environment set-up, you'll be well-equipped to contribute to eth2!
+The Prysm project is a large monorepo containing all sorts of tools and services that implement the eth2 protocol. We use the Bazel build system for everything we do in development, helping everyone have reproducible builds. If you want to build the whole project, you can run the following command:
```

Comment:
This says bazel but then has a go command right below, maybe mention support go as well somehow?

Reply:
I see we have similar text below, would be nice to make this more distinct then.